src/app/clusters/wifi-network-management-server/wifi-network-management-server.h

```diff
@@ -0,0 +1,75 @@
+/**
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/AttributeAccessInterface.h>
+#include <app/CommandHandlerInterface.h>
+#include <crypto/CHIPCryptoPAL.h>
+#include <lib/core/CHIPError.h>
+#include <lib/core/Global.h>
+#include <lib/support/Span.h>
+
+void emberAfWiFiNetworkManagementClusterServerInitCallback(chip::EndpointId);
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace WiFiNetworkManagement {
+
+class Server : private AttributeAccessInterface, private CommandHandlerInterface
```

Comment:
nit: Would you consider merging namespace and class name into `WiFiNetworkManagementServer` ?

Reasoning being that `Server` is a generic name and if this pattern gets repeated, grepping for `Server` will start having a lot of hits.

Reply:
Yes, I think that's much better... (I was trying to follow an existing pattern here but we should probably standardize on something like `<cluster name>Server` instead)

Reply:
Done